packages/tse-backend/src/modules/trading-strategy/strategies/market-making/market-making.controller.ts

```diff
@@ -0,0 +1,62 @@
+import { ApiTags } from '@nestjs/swagger';
+import { Body, Controller, Post, Query } from '@nestjs/common';
+import { InjectMapper } from '@automapper/nestjs';
+import { Mapper } from '@automapper/core';
+import {
+  MarketMakingStrategyActionCommand,
+  MarketMakingStrategyActionDto,
+  MarketMakingStrategyCommand,
+  MarketMakingStrategyDto,
+} from './model/market-making.dto';
+import { MarketMakingStrategy } from './market-making.strategy';
+
+@ApiTags('trading-strategy')
+@Controller('arbitrage')
+export class MarketMakingController {
+  constructor(
+    private readonly service: MarketMakingStrategy,
+    @InjectMapper() private readonly mapper: Mapper,
+  ) {}
+
+  @Post('/create-market-making')
+  async createMarketMakingStrategy(@Body() dto: MarketMakingStrategyDto) {
+    const command = this.mapper.map(
+      dto,
+      MarketMakingStrategyDto,
+      MarketMakingStrategyCommand,
+    );
+    return await this.service.create(command);
+  }
+
+  @Post('/pause-market-making')
+  async pauseMarketMakingStrategy(@Query() dto: MarketMakingStrategyActionDto) {
+    const command = this.mapper.map(
+      dto,
+      MarketMakingStrategyActionDto,
+      MarketMakingStrategyActionCommand,
+    );
+    return await this.service.pause(command);
+  }
+
+  @Post('/stop-market-making')
```

Comment:
Should that be POST?

Reply:
What's the diffrence between: Pause & Stop?

Reply:
The Pause method stops the strategy execution (clear the interval),
The Stop method stops the strategy execution and cancels active (unfilled) orders

Reply:
Fixed: 04b08ce7